src/components/picker/PickerItemsList.tsx

```diff
@@ -124,19 +123,29 @@ const PickerItemsList = (props: PickerItemsListProps) => {
     );
   };
 
-  const renderWheel = () => {
-    const {cancelButtonProps, cancelLabel, doneLabel, title, titleStyle, containerStyle, useSafeArea} =
-      topBarProps ?? {};
-
-    return (
-      <View useSafeArea={useSafeArea}>
+  const renderPickerHeader = () => {
+    if (renderHeader) {
+      return renderHeader?.({onDone: topBarProps?.onDone, onCancel: topBarProps?.onCancel});
+    } else if (useWheelPicker) {
+      const {cancelButtonProps, cancelLabel, doneLabel, title, titleStyle, containerStyle} = topBarProps ?? {};
+      return (
         <View row spread padding-page style={containerStyle}>
           {(cancelButtonProps || cancelLabel) && renderCancel()}
           <Text style={titleStyle}>{title}</Text>
           <Text text70 $textPrimary accessibilityRole={'button'} onPress={() => context.onPress(wheelPickerValue)}>
             {doneLabel ?? 'Select'}
           </Text>
         </View>
+      );
+    } else if (!useDialog || mode === PickerModes.MULTI) {
+      return <Modal.TopBar testID={`${props.testID}.topBar`} {...topBarProps}/>;
+    }
+  };
+
+  const renderWheel = () => {
+    const {useSafeArea} = topBarProps ?? {};
```

Comment:
why not get it directly `useSafeArea={topBarProps?.useSafeArea}` instead of defaulting to empty object?